manifests/init.pp

```diff
@@ -121,7 +121,7 @@
 #
 class simp (
   Hash                            $scenario_map,
-  Enum['simp', 'simp_lite', 'poss', 'none'] $scenario         = 'simp',
+  Enum['simp', 'simp_lite', 'poss', 'idm', 'none'] $scenario  = 'simp',
```

Comment:
I don't think so, ideally this should be a String since the code itself will throw a fail if you use a scenario that doesn't exist in the scenario data, and it will allow us to add scenarios (and allow users to create scenarios) easier.